src/elements/content-preview/preview-header/PreviewHeader.js

```diff
@@ -64,82 +64,82 @@ const PreviewHeader = ({
     const drawMsg = intl.formatMessage(messages.drawAnnotation);
     const pointMsg = intl.formatMessage(messages.pointAnnotation);
 
-    const className = classNames('bcpr-header', {
-        'bcpr-header--basic': !isPreviewingCurrentVersion,
-    });
-
     return (
-        <div className={className}>
-            <div className="bp-header bp-base-header">
-                <FileInfo file={file} version={selectedVersion} />
-                <div className="bcpr-btns">
-                    {shouldRenderOpenWith && isPreviewingCurrentVersion && (
-                        <LoadableContentOpenWith
-                            className="bcpr-bcow-btn"
-                            fileId={fileId}
-                            token={token}
-                            {...contentOpenWithProps}
-                        />
-                    )}
-                    {canAnnotate && isPreviewingCurrentVersion && (
-                        <React.Fragment>
+        <header
+            className={classNames('bcpr-PreviewHeader', {
+                'bcpr-PreviewHeader--basic': !isPreviewingCurrentVersion,
+            })}
+        >
+            <FileInfo file={file} version={selectedVersion} />
+
+            <div className="bcpr-PreviewHeader-controls">
+                {isPreviewingCurrentVersion && (
+                    <React.Fragment>
```

Comment:
thoughts about `<React.Fragment>` versus using `<>`? I prefer the former, personally.

Reply:
I prefer the former, too. I don't think we have a standard for it.